infra/discovery/src/listener/listener.js

```diff
@@ -125,6 +125,13 @@ async function main() {
   const tickIntervalSeconds = 5
   let start
 
+  /**
+   * Make sure we're using the block number from the latest retrieved event.  This is set to false
+   * in the graphql contracts initialization that's being used here.
+   */
+  contractsContext.marketplace.eventCache.useEventAsLatestBlock = true
```

Comment:
Any reason we should not make this the default ?
The DApp could probably benefit from not skipping events as well ?

Reply:
If there are no objections, I'll make this change.  It shouldn't cause any more requests, just open up the request window a bit(potential load for infura/JSON-RPC).

Reply:
I've gone ahead and made this change.  It simplified things a little.